lib/taurus/core/taurusmanager.py

```diff
@@ -82,13 +82,16 @@ def reInit(self):
         this_path = os.path.abspath(__file__)
         self._this_path = os.path.dirname(this_path)
         self._serialization_mode = self.DefaultSerializationMode
-        if self._serialization_mode == TaurusSerializationMode.Concurrent:
-            self._thread_pool = ThreadPool(name="TaurusTP",
-                                           parent=self,
-                                           Psize=5,
-                                           Qsize=1000)
-        else:
-            self._thread_pool = None
+
+        self._thread_pool = ThreadPool(name="TaurusTP",
+                                       parent=self,
+                                       Psize=5,
+                                       Qsize=1000)
+
+        self._sthread_pool = ThreadPool(name="TaurusTSP",
+                                       parent=self,
+                                       Psize=1,
+                                       Qsize=float("inf")) #TODO check value
```

Comment:
Can you put a float there?. Also, PEP8: add extra space before and after `#`

Reply:
I fixed the TODO